maze_transformer/generation/latticemaze.py

```diff
@@ -290,6 +291,24 @@ def find_shortest_path(
                 g_score[neighbor] = g_temp
                 f_score[neighbor] = g_score[neighbor] + self.heuristic(neighbor, c_end)
 
+    def get_nodes(self) -> list[Coord]:
+        """return a list of all nodes in the maze"""
+
+        return [
+            (row, col)
+            for row in range(self.grid_shape[0])
+            for col in range(self.grid_shape[1])
+        ]
```

Comment:
nit: does order matter? If not, should this be a set?

Reply:
I think you're right, order does not matter here. I think if we ever care about order in future, we would probably want a seperate method that returns a 2D array of nodes.

Reply:
Okay I tried using a set and discovered that sampling from a set is deprecated in python 3.9 and removed as of 3.11. So this actually needs to be a list.